client/src/leap/soledad/client/api.py

```diff
@@ -70,6 +71,26 @@
 """
 SOLEDAD_CERT = None
 
+# A whitelist of modules from where to collect plugins dynamically.
+# For the moment restricted to leap namespace, but the idea is that we can pass
+# other "trusted" modules as options to the initialization of soledad.
```

Comment:
@andrejb we've talk about that with @kalikaneko, the problem is that you can not register the plugins after start soledad, because you don't want to have any sync happening before having your plugins attached to soledad. That means that you'll need to tell soledad in init time the list of plugins, and that will mean that bitmask will need to know it and pass it to soledad.

We though that was interesting to don't require bitmask to know who is hooking to the soledad sync. I had some doubts if this solution is not over-ingeneer it, but I think I like it more than passing a list of post-sync-hooks in the initialization of soledad.